extending-dummy4j.md

```diff
@@ -41,20 +41,29 @@ If you choose to, you can place all of your definitions in one file. You can als
 
 ## Parser
```

Comment:
There is no `Parser` in the codebase. Is the documentation about 'ExpressionResolver'? Since this part is about extending Dummy4j it could be useful to hint what exactly should be replaced in code to extend this behaviour.

Reply:
It is. It might be a good idea to unify the naming here.